src/inspect_ai/_view/www/src/App.mjs

```diff
@@ -254,18 +253,22 @@ export function App() {
       setSelected(0);
     }
 
+    new ClipboardJS(".clipboard-button,.copy-button");
+
     // poll every 1s for events
-    setInterval(() => {
-      api.client_events().then(async (events) => {
-        if (events.includes("reload")) {
-          window.location.reload(true);
-        }
-        if (events.includes("refresh-evals")) {
-          await load();
-          setSelected(0);
-        }
-      });
-    }, 1000);
+    if (pollForLogs) {
```

Comment:
I don't see where this is enabled - are ever polling for logs?

Reply:
I added this because I don't always want to poll for logs:

1. My use case is sharing non-changing content from somewhere other than the local filesystem, so polling is pointless and introduces additional auth issues for me. In retrospect maybe it's better to implement `client_events` to just do nothing, but this is a bit more explicit perhaps?
2. It was spamming my dev tools console while I was trying to preserve http logs and I got mad at it 😅

Happy to walk this back and just implement `client_events` in my API to do things instead?

Reply:
This also makes sense - I missed this:

`export function App({ api, pollForLogs = true }) {...`